demo/misc.constants

```diff
@@ -1,8 +1,6 @@
 {
-  "cromwell_host": "http://cromwell.example.com:8000",
```

Comment:
Why remove the `cromwell_host` argument here?

Reply:
It's not longer used in any of the demo olives.